test/e2e/main_test.go

```diff
@@ -344,6 +344,32 @@ func TestManualMeteringInstall(t *testing.T) {
 			},
 			MeteringConfigManifestFilename: "s3.yaml",
 		},
+		{
+			Name:                      "ValidateNFS-ReportDynamicInputData",
+			MeteringOperatorImageRepo: meteringOperatorImageRepo,
+			MeteringOperatorImageTag:  meteringOperatorImageTag,
+			Skip:                      !runAllInstallTests,
+			PreInstallFunc:            createNFSProvisioner,
+			InstallSubTests: []InstallTestCase{
+				{
+					Name:     "testReportingProducesData",
+					TestFunc: testReportingProducesData,
+					ExtraEnvVars: []string{
+						"REPORTING_OPERATOR_PROMETHEUS_DATASOURCE_MAX_IMPORT_BACKFILL_DURATION=15m",
+						"REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_INTERVAL=30s",
+						"REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_CHUNK_SIZE=5m",
+						"REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_INTERVAL=5m",
+						"REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_STEP_SIZE=60s",
+					},
+				},
+				{
+					Name:         "testEnsureS3BucketIsDeleted",
+					TestFunc:     testEnsureS3BucketIsDeleted,
+					ExtraEnvVars: []string{},
+				},
```

Comment:
Not need to run this post-install tests as no s3 bucket has been configured for metering.